android/app/build.gradle

```diff
@@ -58,12 +69,7 @@ android {
     release {
       minifyEnabled true
       proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-      zipAlignEnabled true
```

Comment:
what happened to this step?

Reply:
[The docs indicate signed apps are also zipaligned](https://developer.android.com/studio/publish/app-signing#gradle-sign)
> Now, when you build your app from the command line by invoking a Gradle task, Gradle signs your app (and runs zipalign) for you.